PencilCase/PencilCase/scripts/application.ts

```diff
@@ -109,12 +113,25 @@ export class Application {
                                         CreatedDate datetime,\
                                         ModifiedDate datetime,\
                                         foreign key(RetailUnit) references UnitOfMeasure(Id),\
-                                        foreign key(WholesaleUnit) references UnnitOfMeasure(Id)\
+                                        foreign key(WholesaleUnit) references UnitOfMeasure(Id)\
+                                        )', [], null, this.onDBError);
+
+                // I have to name the table as Orders instead of Order, because Order is a reserved word by SQL Lite, it did took me some time
+                // https://www.sqlite.org/lang_keywords.html
+                //transaction.executeSql('DROP TABLE IF EXISTS Orders', [], null, this.onDBError);
+                transaction.executeSql('CREATE TABLE Orders (\
```

Comment:
Create "Orders" table. 
Note that it's "Orders" but not "Order", since "Order" is a reserved key word for SQL Lite. Maybe the order by syntax.